test/service/test_response.py

```diff
@@ -1170,6 +1177,30 @@ def test_cell_suspension_response(self):
         cell_suspension = one(keyword_response['hits'][0]['cellSuspensions'])
         self.assertEqual(["Plasma cells"], cell_suspension['selectedCellType'])
 
+    def test_filter_by_projectId(self):
+        """
+        Test response when using a projectId filter
+        """
+        testdata = [
+            {
+                'id': '627cb0ba-b8a1-405a-b58f-0add82c3d635',
+                'title': '10x 1 Run Integration Test'
+            },
+            {
+                'id': '250aef61-a15b-4d97-b8b4-54bb997c1d7d',
+                'title': 'Bone marrow plasma cells from hip replacement surgeries'
+            }
+        ]
+        for data in testdata:
+            url = self.base_url + "/repository/projects?size=15" \
```

Comment:
```suggestion
            url = self.base_url + "/repository/projects?size=2" \
```

Reply:
Cover `/repository/files` and `/repository/specimens`. And use `with self.subTest(entity_type=entity_type)`